src/locales/en.json

```diff
@@ -82,8 +82,9 @@
         "delete_playlist_confirm": "Are you sure you want to delete this playlist?",
         "please_select_playlist": "Please select a playlist",
         "delete_account": "Delete Account",
-        "logout": "Logout",
-        "minimize_recommendations_default": "Minimize Recommendations by default"
+        "logout": "Log out this device",
+        "minimize_recommendations_default": "Minimize Recommendations by default",
+        "invalidate_session": "Log out all devices"
```

Comment:
I would have "Logout" as one word and, and add "from" after it. Just a nit-pick :p